db/migrate/20150315001756_add_private_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrivateLinks < ActiveRecord::Migration
+  def change
+    add_column :links, :private_links, :string
```

Comment:
For det første skal dette være en "boolean". En boolean er en værdi der kun kan være `true` eller `false` (og `nil`/tom men det er det samme som `false` i næsten alle sammenhænge i ruby).

Så lav en rollback og ændre dette til `add_column :links, :private, :boolean, null: false, default: false` (ingen grund til at kalde det `private_link`, link er indforstået plus at nu når du har et link i en variable kan du skrive `@link.private?`)